src/cascadia/TerminalApp/App.cpp

```diff
@@ -254,6 +292,37 @@ namespace winrt::TerminalApp::implementation
             settingsItem.Click({ this, &App::_SettingsButtonOnClick });
             newTabFlyout.Items().Append(settingsItem);
 
+            // Using alternate method here than VirtualKey due to https://github.com/microsoft/microsoft-ui-xaml/issues/708
+            // this should really use KeyboardAccelerator API bug above bug in framework prevents it, using override
+            auto settingsKeyChord = _keyBindings.GetKeyBinding(ShortcutAction::OpenSettings);

```

Comment:
This could probably be lifted out as `SetAcceleratorForMenuItem(FlyoutMenuItem&, const KeyChord&)`

Reply:
Would you just put this as a local private method or yet another static?

Reply:
if it doesn't require any state from `this`, my inclination is to make it a private static. @zadjii-msft may think otherwise?

Reply:
Yea I agree, a private static is probably the right call here.

Reply:
I put the other statics in AppKeyBindings...kind of felt that App object was piling up and that felt like the right place...same for this one as well?

Reply:
I think right now App is the only (or one of the only) components at this level that knows about UI things... which absolutely has its advantages and disadvantages. We'll need to come through here and split things up, especially once we get a xaml root page and stop constructing our UI ourselves...
until then, it'll be fine to keep it in App